pep-0639.rst

```diff
@@ -53,8 +52,8 @@ The PEP also:
   :ref:`other ecosystems <639-license-doc-other-projects>`.
 
 The changes in this PEP will update the
-:`core metadata <coremetadataspec_>`__ to version 2.3, modify the
-`PEP 621 project metadata specification <pep621spec_>`__,
+:`core metadata <coremetadataspec_>`__ to version 2.4, modify the
```

Comment:
```suggestion
`core metadata <coremetadataspec_>`__ to version 2.4, modify the
```

Reply:
Rebased in to fix, thanks—that's been there for a while.